homeassistant/components/tuya/switch.py

```diff
@@ -520,6 +520,20 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Thermostat
+    # https://developer.tuya.com/en/docs/iot/f?id=K9gf45ld5l0t9
+    "wk": (
+        SwitchEntityDescription(
+            key=DPCode.SWITCH,
+            translation_key="on_off",
+            entity_category=EntityCategory.CONFIG,
+        ),
```

Comment:
The on/off handling is already handled by the climate entity:

<https://github.com/home-assistant/core/blob/04c8a5574a2041885e072562315f5f55317e659e/homeassistant/components/tuya/climate.py#L224-L228>

And thus should not be added as a switch.